### ISSUE TITLE
approve() interface for the Archive extension

### ISSUE THREAD

**Issue by ferencdg** [Toniq-Labs/extendable-token#13]
Ethereum emits a new event every time a new transaction is done or an approval happened. Currently in the Archive interface we are only able to retrieve transaction events, but not approval events. Should we also archive the approval events?

**Comment by stephenandrews** [comment]
I agree, we want to be able to log every interaction. The Archive extension still needs a lot more work, and will be the focus once our initial release is complete (marketplace integration)
